otx/algorithms/action/adapters/mmaction/task.py

```diff
@@ -0,0 +1,525 @@
+
+        if self._model_ckpt is not None:
+            recipe_cfg.load_from = self._model_ckpt
+
+        self._config = recipe_cfg
+        return recipe_cfg
+
+    # pylint: disable=too-many-branches, too-many-statements
+    def _train_model(
+        self,
+        dataset: DatasetEntity,
+    ):
+        """Train function in MMActionTask."""
+        logger.info("init data cfg.")
+        self._data_cfg = ConfigDict(data=ConfigDict())
+
+        for cfg_key, subset in zip(
+            ["train", "val", "unlabeled"],
+            [Subset.TRAINING, Subset.VALIDATION, Subset.UNLABELED],
+        ):
+            subset = get_dataset(dataset, subset)
+            if subset and self._data_cfg is not None:
+                self._data_cfg.data[cfg_key] = ConfigDict(
+                    otx_dataset=subset,
+                    labels=self._labels,
+                )
+
+        self._is_training = True
+
+        self._init_task()
+
+        cfg = self.configure(True, "train", None)
+        logger.info("train!")
+
+        timestamp = time.strftime("%Y%m%d_%H%M%S", time.localtime())
+
+        # Environment
+        logger.info(f"cfg.gpu_ids = {cfg.gpu_ids}, distributed = {cfg.distributed}")
+        env_info_dict = collect_env()
+        env_info = "\n".join([(f"{k}: {v}") for k, v in env_info_dict.items()])
+        dash_line = "-" * 60 + "\n"
+        logger.info(f"Environment info:\n{dash_line}{env_info}\n{dash_line}")
+
+        # Data
+        datasets = [build_dataset(cfg.data.train)]
+
+        # FIXME: Currently action do not support multi batch evaluation. This will be fixed
+        if "val" in cfg.data:
+            cfg.data.val_dataloader["videos_per_gpu"] = 1
+
+        # Target classes
+        if "task_adapt" in cfg:
+            target_classes = cfg.task_adapt.get("final", [])
+        else:
+            target_classes = datasets[0].CLASSES
+
+        # Metadata
+        meta = dict()
+        meta["env_info"] = env_info
+        meta["seed"] = cfg.seed
+        meta["exp_name"] = cfg.work_dir
+        if cfg.checkpoint_config is not None:
+            cfg.checkpoint_config.meta = dict(
+                mmdet_version=__version__ + get_git_hash()[:7],
+                CLASSES=target_classes,
+            )
+
+        # Model
+        model = self.build_model(cfg, fp16=cfg.get("fp16", False))
+        model.train()
+        model.CLASSES = target_classes
+
+        if cfg.distributed:
+            torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)
+
+        validate = bool(cfg.data.get("val", None))
+        train_model(
+            model,
+            datasets,
+            cfg,
+            distributed=cfg.distributed,
+            validate=validate,
+            timestamp=timestamp,
+            meta=meta,
+        )
+
+        # Save outputs
+        output_ckpt_path = os.path.join(cfg.work_dir, "latest.pth")
+        best_ckpt_path = glob.glob(os.path.join(cfg.work_dir, "best_*.pth"))
+        if len(best_ckpt_path) > 0:
```

Comment:
```suggestion
        if best_ckpt_path:
```
it doesn't matter, it's recommended way to check empty list